rllib/offline/estimators/tests/test_ope.py

```diff
@@ -42,22 +37,16 @@ def setUpClass(cls):
             .resources(num_gpus=int(os.environ.get("RLLIB_NUM_GPUS", 0)))
             .offline_data(input_=train_data)
             .evaluation(
-                evaluation_interval=None,
```

Comment:
with `evaluation_interval = 1` you are running the in-training evaluation inside the setup method. My suggestion is to separate in-training from stand-alone evaluation.

Reply:
Moved to the same function `test_ope_in_algo`